dockerfiles/run/guice/destination/conf/elasticsearch.properties

```diff
@@ -26,3 +26,8 @@ elasticsearch.nb.shards=1
 elasticsearch.nb.replica=0
 elasticsearch.retryConnection.maxRetries=7
 elasticsearch.retryConnection.minDelay=3000
+
+# This properties defines the maximal length of search results. Higher values
+# means more results but costs more performances.
+# Default value is 100
+elasticsearch.search.size=100
```

Comment:
Thank you very much for the explanation !

What bothers me is that I won't be able to found the mail I want when I need it.
From a business point of view, it's not ok to not be able to find a given email, even if there are much results.
So if we need scroll, let's do it.